financialaid/constants.py

```diff
@@ -0,0 +1,224 @@
+"""
+Constants for financialaid
+"""
+
+
+DEFAULT_INCOME_THRESHOLD = 100000
+COUNTRY_INCOME_THRESHOLDS = {
```

Comment:
@pdpinch this data will all be exposed in the source code, is that fine with you?